web/cobrands/sass/_mixins.scss

```diff
@@ -12,6 +12,11 @@ $button-margin-bottom: 0 !default;
 $button-margin-right: 0 !default;
 $button-margin-left: 0 !default;
 
+// Focus Outline
```

Comment:
These seem very generically named if they're only being used for the focus within `.alerts__nearby-activity__photos`. Are these likely to be overridden? I wouldn't think so, so I'm not sure it's worth having these as variables until such a time as that proves necessary.